### PR TITLE
fixed #4259

### LINKED ISSUE #4259: [New game]: text not visible
### 🎮 Game Request

text is not visible in light mode

### Point down the features

Screenshot 2024-06-05 230129
Screenshot 2024-06-05 230223

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by dnyandeepchute**
## PR Description 📜
text was not visible in light mode
Fixes #4259

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

before:
Screenshot 2024-06-06 124131
Screenshot 2024-06-06 124140

after:
Screenshot 2024-06-06 124103
Screenshot 2024-06-06 124117

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @dnyandeepchute ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please create a new folder inside the "Games" folder.
  - Please add a README.md file inside the newly created folder.
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Comment by dnyandeepchute** [comment]
@kunjgit it was a bug, please review it.

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please create a new folder inside the "Games" folder.
  - Please add a README.md file inside the newly created folder.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please create a new folder inside the "Games" folder.
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please create a new folder inside the "Games" folder.
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Comment by dnyandeepchute** [comment]
@kunjgit please merge this, thank you.

File: Games/demo/style.css
Comment (ishita-43) [inline Games/demo/style.css:1]: delete this file if not needed!!

File: Games/demo/index.html
Comment (ishita-43) [inline Games/demo/index.html:1]: delete this file if not needed!!

File: Games/demo/README.md
Comment (ishita-43) [inline Games/demo/README.md:1]: This file is not nedded !! Remove this one..

**Review by ishita-43 — CHANGES_REQUESTED** [review]
You dont need extra files for bug correction remove those files !!
Hoping you will make changes soon!!

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @dnyandeepchute,
You need to make the following changes:
  - Please create a new folder inside the "Games" folder.
  - Please add a README.md file inside the newly created folder.

Hoping that you will make those changes soon 🚀

**Comment by dnyandeepchute** [comment]
done please check.

File: Games/demo/README.md
Comment (dnyandeepchute) [reply Games/demo/README.md:1]: done please check.

File: Games/demo/index.html
Comment (dnyandeepchute) [reply Games/demo/index.html:1]: done please check.

File: Games/demo/style.css
Comment (dnyandeepchute) [reply Games/demo/style.css:1]: done please check.

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by dnyandeepchute** [comment]
@ishita-43 it was for level3, is it??

**Comment by ishita-43** [comment]
> @ishita-43 it was for level3, is it??

YOU DID bug removal right!!
If its the case we place bugs removal in level1

**Comment by kunjgit** [comment]
Thank you  @dnyandeepchute , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**